src/content/services/monetizationTagManager.ts

```diff
@@ -69,12 +70,18 @@ export class MonetizationTagManager extends EventEmitter {
     this.monetizationTags.forEach((tagDetails, tag) => {
       if (tagDetails.requestId !== requestId) return
 
-      const customEvent = new CustomEvent('monetization', {
-        bubbles: true,
-        detail: mozClone(details, this.document)
-      })
-
-      tag.dispatchEvent(customEvent)
+      const { incomingPayment, paymentPointer, receiveAmount } = details
+      const amountSent: PaymentCurrencyAmount = {
+        currency: receiveAmount.assetCode,
+        value: transformBalance(receiveAmount.value, receiveAmount.assetScale)
```

Comment:
Not really important, but I would prefer sending the correct details directly from background. Thoughts?

Reply:
Oh yes, I didn't realise I was in content/